src/WebJobs.Script/ScriptHostBuilderExtensions.cs

```diff
@@ -256,10 +255,10 @@ internal static void ConfigureApplicationInsights(HostBuilderContext context, IL
             }
         }
 
-        internal static ExtensionBundleOptions GetExtensionBundleOptions(HostBuilderContext context, AspNetCore.Hosting.IHostingEnvironment hostingEnvironment)
+        internal static ExtensionBundleOptions GetExtensionBundleOptions(HostBuilderContext context)
         {
             var options = new ExtensionBundleOptions();
-            var optionsSetup = new ExtensionBundleOptionsSetup(context.Configuration, SystemEnvironment.Instance, hostingEnvironment);
+            var optionsSetup = new ExtensionBundleOptionsSetup(context.Configuration, SystemEnvironment.Instance, context.HostingEnvironment);
```

Comment:
👍

Reply:
It's a bit confusing that this is an `IOptionsSetup<ExtensionBundleOptions>` but not really used as one (not registered, just explicitly used here). I can see that confusing others if they're working on this code. We might want to just drop that implementation and rename this to reflect that this is a configuration helper.